django_socio_grpc/tests/test_ordering_request_struct.py

```diff
@@ -0,0 +1,116 @@
+        self.old_pagination_class = UnitTestModelWithStructFilterService.pagination_class
+        UnitTestModelWithStructFilterService.pagination_class = None
+
+        UnitTestModel(title="zzzz", text=text).save()
+        self.fake_grpc = FakeFullAIOGRPC(
+            add_UnitTestModelWithStructFilterControllerServicer_to_server,
+            UnitTestModelWithStructFilterService.as_servicer(),
+        )
+
+    def tearDown(self):
+        UnitTestModelWithStructFilterService.pagination_class = self.old_pagination_class
+
+    @override_settings(
+        GRPC_FRAMEWORK={
+            "GRPC_ASYNC": True,
+            "FILTER_BEHAVIOR": FilterAndPaginationBehaviorOptions.REQUEST_STRUCT_STRICT,
+        }
+    )
+    async def test_django_ordering_with_struct_request(self):
+        grpc_stub = self.fake_grpc.get_fake_stub(UnitTestModelWithStructFilterControllerStub)
+        filter_as_dict = {"ordering": "title"}
+        filter_as_struct = struct_pb2.Struct()
+        filter_as_struct.update(filter_as_dict)
+        request = UnitTestModelWithStructFilterListRequest(_filters=filter_as_struct)
+        response = await grpc_stub.List(request=request)
+
+        self.assertEqual(len(response.results), 11)
+        # responses_as_list[0] is type of django_socio_grpc.tests.grpc_test_utils.unittest_pb2.Test
+        self.assertEqual(response.results[0].title, "z")
+        self.assertEqual(response.results[1].title, "zz")
+        self.assertEqual(response.results[2].title, "zzz")
+        self.assertEqual(response.results[3].title, "zzzz")
+
+        filtered_models = UnitTestModel.objects.all().order_by("-title")
+        print((await filtered_models.afirst()).title)
+
+        # Testing metadata filter not working
+        grpc_stub = self.fake_grpc.get_fake_stub(UnitTestModelWithStructFilterControllerStub)
+        filter_as_dict = {"ordering": "-title"}
+        filter_as_struct = struct_pb2.Struct()
+        filter_as_struct.update(filter_as_dict)
+        request = UnitTestModelWithStructFilterListRequest(_filters=filter_as_struct)
+        response = await grpc_stub.List(request=request)
+
+        self.assertEqual(len(response.results), 11)
+        # responses_as_list[0] is type of django_socio_grpc.tests.grpc_test_utils.unittest_pb2.Test
+        self.assertEqual(response.results[0].title, "zzzzzzzzzz")
+        self.assertEqual(response.results[1].title, "zzzzzzzzz")
+        self.assertEqual(response.results[2].title, "zzzzzzzz")
+        self.assertEqual(response.results[3].title, "zzzzzzz")
+
+    @override_settings(
+        GRPC_FRAMEWORK={
+            "GRPC_ASYNC": True,
+            "FILTER_BEHAVIOR": FilterAndPaginationBehaviorOptions.REQUEST_STRUCT_STRICT,
+        }
+    )
+    async def test_django_ordering_array_with_struct_request(self):
+        grpc_stub = self.fake_grpc.get_fake_stub(UnitTestModelWithStructFilterControllerStub)
+        filter_as_dict = {"ordering": ["title"]}
```

Comment:
You are not testing multiple ordering at the same time is it volunteer?

Reply:
Yes because this is a behavior tested in DRF directly.
Also I need to create an other dataset to make it coherent.
But if you prefer one I can do it.

Reply:
Either create another dataset or make a comment that the behavior is tested in DRF with a link to where it is tested. Coming from someone that doesn't know DRF I'm missing information just by reading the code